config/routes.rb

```diff
@@ -1,10 +1,18 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
+
+  get '/home' => 'pages#home'
```

Comment:
This should have been:

```ruby
get '/' => 'pages#home'
```

Please change by the syntax that doesn't use hash rockets `=>` 
And change it 😄